plugins/kubernetes-backend/examples/dice-roller/README.md

```diff
@@ -2,48 +2,35 @@
 
 This can be used to run the kubernetes plugin locally against a mock service.
 
-# Viewing in local Minikube running Backstage locally
+# Viewing in local Kind running Backstage locally
 
 ## Prerequisites
 
 - kubectl installed
-- Minikube installed, with the following addons
-  - metrics-server
-  - ingress
+- Kind installed
```

Comment:
I'm thinking a link here would be nice tbh 😅

Reply:
done